internal/infrastructure/kubernetes/bootstrap.yaml.tpl

```diff
@@ -29,7 +29,7 @@ dynamic_resources:
       set_node_on_first_message_only: true
 static_resources:
   clusters:
-  - connect_timeout: 1s
+  - connect_timeout: 10s
```

Comment:
if we are using 10s here, can we make it consistent with dynamic cluster value, and set that to 10s as well (currently its 5s https://github.com/envoyproxy/gateway/blob/6ff1758ecd293255fd2815b7bdb7272f1c758c33/internal/xds/translator/cluster.go#L36) ?

Reply:
resolved